package.json

```diff
@@ -13,11 +13,16 @@
 	"author": "The WordPress Contributors",
 	"license": "GPL-2.0-or-later",
 	"devDependencies": {
+		"@wordpress/babel-preset-default": "4.3.0",
 		"@wordpress/custom-templated-path-webpack-plugin": "1.2.0",
+		"@wordpress/e2e-test-utils": "2.1.0",
 		"@wordpress/library-export-default-webpack-plugin": "1.1.0",
+		"@wordpress/scripts": "3.3.0",
 		"autoprefixer": "9.4.7",
+		"babel-jest": "24.8.0",
 		"check-node-version": "3.2.0",
 		"copy-webpack-plugin": "^4.6.0",
+		"core-js": "3.1.4",
```

Comment:
@wordpress/babel-preset-default babel-jest and core-js dependencies are required to make sure we can use "import" and ESnext features while writing tests, I think this should be absorbed by @wordpress/scripts (cc @gziolo)

Reply:
Yes, this needs to be further explored to ensure that `wp-scripts test-e2e` and `wp-scripts test-unit` work out of the box after you install `@wordpress/scripts`. 

I see `babel-jest` wasn't included :(
`core-js` is tricky as it isn't a direct dependency but we need to find ways to ensure it's there so tests are properly transpiled.

It seems like `@wordpress/babel-preset-default` was necessary only because it's referenced directly or because you had to override configs. We need to fix the code duplication somehow.

Reply:
I'm trying this https://github.com/WordPress/gutenberg/pull/16259.